receiver/hostmetricsreceiver/internal/scraper/networkscraper/network_others.go

```diff
@@ -30,3 +34,7 @@ var allTCPStates = []string{
 	"SYN_RECEIVED",
 	"TIME_WAIT",
 }
+
+func scrapeAndAppendConntrackMetrics(s *scraper, metrics pdata.MetricSlice) error {
+	return nil
```

Comment:
Should we instead return an error to indicate that this is unsupported on other platforms?